runtime/queries/metricsview_comparison_toplist.go

```diff
@@ -237,8 +245,10 @@ func (q *MetricsViewComparison) buildMetricsTopListSQL(mv *runtimev1.MetricsView
 				return "", nil, err
 			}
 			selectCols = append(selectCols, fmt.Sprintf("%s as %s", expr, safeName(m.Name)))
+			labelCols = append(labelCols, fmt.Sprintf("%s as %s", safeName(m.Name), labelMap[m.Name]))
```

Comment:
Doesn't it need `safeName` for the label? (The label can have arbitrary characters, including spaces and quotes)